### ISSUE TITLE
eg5 fails when making seed mechanism

### ISSUE THREAD

**Issue by goldmanm** [ReactionMechanismGenerator/RMG-Py#1115]
I was trying to `make eg5` on the most recent master, and got the following error. Which happened on my local machine and pharos. The [travis test](https://travis-ci.org/ReactionMechanismGenerator/RMG-tests/builds/258905114) for the seed mechanisms PR did not error. Is anyone able to replicate this error? 

```
Making seed mechanism...
trying to find duplicates
NB. the entries have not been renumbered, so these indices are missing.
Traceback (most recent call last):
  File "rmg.py", line 162, in <module>
    rmg.execute(**kwargs)
  File "/home/goldmanm/repos/RMG-Py/rmgpy/rmg/main.py", line 583, in execute
    self.makeSeedMech(firstTime=True)
  File "/home/goldmanm/repos/RMG-Py/rmgpy/rmg/main.py", line 892, in makeSeedMech
    thermoLibrary.save(os.path.join(seedDir, name + '.py'))
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

**Comment by mliu49** [comment]
Apparently, eg5 on RMG-tests is different from eg5 in RMG-Py. Notably, eg5 in RMG-Py is missing the options block. If you add the options block in with the same settings as RMG-tests, then the job runs without erroring. This suggests some problems with default values for the options block.

**Comment by goldmanm** [comment]
Seems like previous issues with arguments causing errors #1067 may be related.

**Comment by mjohnson541** [comment]
closed with #1117
